dts/arm/st/pinctrl_st_stm32.h

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright (c) 2020 Linaro Limited
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef PINCTRL_ST_STM32_H_
+#define PINCTRL_ST_STM32_H_
+
+/* Create a pincfg device tree node:
+ *
+ * The node name and nodelabel will be of the form:
+ *
+ * NODE = <periph>_<signal>_p<port>_<pin>
+ *
+ * NODE: NODE {
+ *     pinmux = <STM32_PINMUX_Z(STM32_DT_PORT_<port>,pin, function)>;
+ *
+ * }
+ *
+ * Also, each macro will fill directly a knwon working pin configuration
+ * depending on the signal that is required.
+ *
+ * So for example:
+ *
+ * DT_STM32_UART_TX(usart1,b,6,AF7);
```

Comment:
The example here is out of sync, order of args needs fixing.

Reply:
Txs, fixed